index.html

```diff
@@ -59,5 +59,170 @@ <h3>LET'S CONNECT</h3>
         </ul>
       </section>
     </main>
+    <!-- Cards section -->
+    <section class="main-container-projects">
+      <div class="cards card-1">
+        <div class="main-img">
+          <img src="./img/Snapshoot Portfolio.png" alt="person" />
+        </div>
+        <article class="second-container">
+          <h2>Tonic</h2>
+          <div class="horizontal-section">
+            <span>CANOPY</span>
+            <img class="bullet-img" src="./img/bullet.svg" alt="bullet" />Back
```

Comment:
- [Optional] Please make sure to add this corner to your headline section
![image](https://user-images.githubusercontent.com/74744787/219783554-20a3fca4-d63b-46b4-8e21-f70b8815615c.png)
![image](https://user-images.githubusercontent.com/74744787/219783586-66bf956a-bdef-4301-8d1b-d5e2daad5e80.png)